cPP_DSC.adoc

```diff
@@ -4019,203 +4015,385 @@ The dependencies between SFRs implemented by the TOE are addressed as follows.
+
+FCS_CKM.5 - (selection-based SFR) included
+
+FCS_CKM.3 is satisfied by FAU_STG_EXT.1 which controls access to keys - included
+
+FCS_CKM.6 - included
 
 |FCS_COP.1/SigVer 
 |[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, or FCS_CKM.5] 
 
 FCS_CKM.6 
-|FCS_CKM.1 and FCS_CKM.6 are required by the PP. FDP_ITC_EXT.1 and FDP_ITC_EXT.2 are also required by the PP and equivalent to FDP_ITC.1 in this case because they all relate to a mechanism by which key data can be imported into the TSF.
+|FDP_ITC.1 is satisfied by FDP_ITC_EXT.1 related to a mechanism by which key data can be imported into the TSF - included
+
+FDP_ITC.2 is satisfied by FDP_ITC_EXT.2 related to a mechanism by which key data can be imported into the TSF - included
+
+FCS_CKM.1 - included
+
+FCS_CKM.5 - (selection-based SFR) included
+
+FCS_CKM.6 - included
+
+FCS_CKM.3 is not necessary as this is only verifying a signature, not generating or storing keys
 
 |FCS_COP.1/SKC 
-|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, or FCS_CKM.5] 
+|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_CKM_EXT.8] 
+
+FCS_CKM.3
 
 FCS_CKM.6 
-|FCS_CKM.1 and FCS_CKM.6 are required by the PP. FDP_ITC_EXT.1 and FDP_ITC_EXT.2 are also required by the PP and equivalent to FDP_ITC.1 in this case because they all relate to a mechanism by which key data can be imported into the TSF.
+
+FCS_OTV_EXT.1
+|FDP_ITC_EXT.1 satisfies FDP_ITC.1 related to a mechanism by which key data can be imported into the TSF - included
+
+FDP_ITC_EXT.2 satisfies FDP_ITC.2 related to a mechanism by which key data can be imported into the TSF - included
+
+FCS_CKM.1 - included
+
+FCS_CKM.5 - (selection-based SFR) included
+
+FCS_CKM_EXT.7 - (selection-based SFR) included
+
+FCS_CKM_EXT.8 - (selection-based SFR) included
+
+FCS_CKM.3 is satisfied by FAU_STG_EXT.1 which controls access to keys - included
+
+FCS_CKM.6 - included
+
+FCS_OTV_EXT.1 - included
 
 |FCS_RBG.1 
-|FCS_COP.1 
-|FCS_COP.1 is required by the PP.
+|[FCS_RBG.2 or FCS_RBG.3]
+
+FPT_FLS.1
+
+FPT_TST.1
+
+|FCS_RBG.2 - (selection-based SFR) included
+
+FCS_RBG.3 - (selection-based SFR) included
+
+FPT_FLS.1 - included
+
+FPT_TST.1 - included
 
 |FCS_OTV_EXT.1 
 |FCS_RBG.1 
-|FCS_RBG.1 is required by the PP.
+|FCS_RBG.1 - included
 
 |FCS_STG_EXT.1 
-|No dependencies. 
+|No dependencies
 |N/A
 
 |FDP_ACC.1 
 |FDP_ACF.1 
-|FDP_ACF.1 is required by the PP.
+|FDP_ACF.1 - included
 
 |FDP_ACF.1 
 |FDP_ACC.1 
 
 FMT_MSA.3
-|FDP_ACC.1 and FMT_MSA.3 are required by the PP.
+|FDP_ACC.1 - included
+
+FMT_MSA.3 - included
 
 |FDP_ETC_EXT.2 
 |FCS_COP.1 
-|FCS_COP.1 is required by the PP.
+|FCS_COP.1 - included
 
 |FDP_FRS_EXT.1 
 |FDP_FRS_EXT.2 
-|FDP_FRS_EXT.2 is an optional SFR in the PP. It does not need to be required because the PP's definition of FDP_FRS_EXT.1 allows the ST author to specify that no factory reset is possible, in which case the behavior described by FDP_FRS_EXT.2 is not triggered.
+|FDP_FRS_EXT.2 - (optional-based SFR) included
```

Comment:
I feel like "optional-based" isn't really the right wording?

Reply:
changed

Reply:
it should actually be selection-based